tests/unit/aegra/test_middleware.py

```
@@ -80,3 +88,57 @@ def test_unknown_auth_type(self):
+        secret = "a" * _MIN_JWT_SECRET_BYTES
+        with (
+            patch("deep_agent.aegra.middleware.AUTH_TYPE", "jwt"),
+            patch("deep_agent.aegra.middleware.JWT_SECRET", secret),
+        ):
+            validate_auth_config()
+
+
+class TestValidateJwtTokenRejectsWeakSecret:
+    def test_empty_secret_raises_auth_error(self):
+        with patch("deep_agent.aegra.middleware.JWT_SECRET", ""):
+            with pytest.raises(AuthError, match="not configured or too short"):
```

Ruff (0.16.1)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)